src/test/resources/inspections/comparingWithMatcher.kt

```diff
@@ -0,0 +1,25 @@
+package inspections
+
+import io.kotest.core.script.test
+import io.kotest.core.spec.style.FunSpec
+import io.kotest.matchers.Matcher
+import io.kotest.matchers.MatcherResult
+import io.kotest.matchers.shouldBe
+
+class MatcherTest : FunSpec({
+
+   test("foo") {
+      val x: Int? = null
+      x shouldBe AlwaysPassMatcher
```

Comment:
Can we add a test for ComparableMatcher too

Reply:
Fixing 👍🏼